tasks/webpack.js

```diff
@@ -152,7 +152,16 @@ module.exports = function(grunt) {
 				}, statsOptions)));
 			}
 			if(typeof storeStatsTo === "string") {
-				grunt.config.set(storeStatsTo, stats.toJson());
+				grunt.config.set(storeStatsTo, stats.toJson(grunt.util._.merge({
+					colors: false,
+					hash: false,
+					timings: false,
+					assets: true,
+					chunks: false,
+					chunkModules: false,
+					modules: false,
+					children: true
+				}, statsOptions)));

```

Comment:
@arthurjamain would it make more sense to refactor this along with lines 139-153, as it has a check for `if(statsOptions) {`?  I'm not exactly sure what `grunt.util._.merge()` will do if `statsOptions` is undefined.